community.md

```diff
@@ -88,6 +88,19 @@ activity is captured in email form for archival purposes.
   <li> <code>github@</code> for all activity on the GitHub repositories {% include mailing_list_links.html list="github" %} </li>
 </ul>
 
+### GitHub Issues
+
+We use [GitHub Issues](https://github.com/apache/arrow/issues) as a way to ask questions and engage with the Arrow developer
```

Comment:
'as a way to ask questions' seems redundant now that we have discussions?

Reply:
Should we add the GitHub issues for all repositories (`arrow-java`, `arrow-go`, ...) ?

Reply:
I think we should be more clear here. We should redirect users to ask questions to GitHub Discussions.  As a new user with a question if I see three options I still would have no idea what is the preferred way. The current phrasing doesn't help in my opinion.
We have suggested in the past to redirect user questions to the mailing list when they were opened as issues. I feel this phrase was already outdated. I would say something in the lines:
> We use GitHub Issues to track the queue of development work. Bugs and feature requests are welcome. For usage type questions we have GitHub Discussions and might redirect usage type issues to open the question there.

Reply:
> and might redirect usage type issues to open the question there.

FWIW, if someone opens an issue that is clearly a usage question, we don't have to ask to open the question as a discussion instead, we (maintainers) should be able to click a button to convert the issue into a discussion/

Reply:
Thanks, wording changed in c5a078c5da86087a1d4d49a4199a69d879065276.

Reply:
New text in ac8107bda5c669905c871440bd8191e7668cda2c which points people to our repos and their issues/discussions.